objects/obj_controller/Alarm_1.gml

```diff
@@ -587,6 +587,7 @@ if (did==1){
     // Ork planets here
     with(obj_star){
         if (planets>0) and (owner==2) and (p_type[1]!="Dead"){
+            //this object does nothing
```

Comment:
if this does nothing, then let's just delete this section

Reply:
it technically acts as a counter by creating a number of objects ,but the actual object has no intrinsic function

Reply:
have altered comment to reflect